tests/test_cms_config.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import Mock
+
+from django.core.exceptions import ImproperlyConfigured
+
+from cms import app_registration
+from cms.models import Page
+from cms.utils.setup import setup_cms_apps
+
+from djangocms_references.cms_config import ReferencesCMSExtension
+from djangocms_references.test_utils.app_1.models import TestModel1, TestModel2
+from djangocms_references.test_utils.app_2.models import TestModel3, TestModel4
+from djangocms_references.utils import supported_models
+
+from .utils import TestCase
+
+
+class AppRegistrationTestCase(TestCase):
+    def test_missing_cms_config(self):
```

Comment:
Would be good to be more explicit either in the test name or a docstring what is improperly configured here. Atm to understand this test I have to look into the code to check that the code expects a `reference_models` attribute.